count-repeat/meta.json

```diff
@@ -0,0 +1,8 @@
+{
+    "title": "CountRepeat",
+    "name": "countRepeat",
+    "difficulty": "Hard",
+    "author": "Jutulli",
+    "category": "lists"
+  }
```

Comment:
the squiggly bracket at the end, make sure its in the same indentation level as the first one.